packages/kotti-ui/source/kotti-filter/KtFilter.vue

```diff
@@ -0,0 +1,219 @@
+<template>
+	<div class="kt-filter">
+		<div class="kt-filter__content">
+			<div
+				v-for="(filter, index) in value"
+				:key="`${filter.key}_${index}`"
```

Comment:
Why was `index` necessary here/how is this benefitial? As far as I can tell, this should either be `:key="filter.key"` or `:key="index"`